stat.go

```diff
@@ -3,15 +3,70 @@ package procfs
 import (
 	"bufio"
 	"fmt"
+	"io"
 	"os"
 	"strconv"
 	"strings"
 )
 
+const (
+	userHz = 100
```

Comment:
The constant is defined in proc_stat as userHZ, I'll re-use that. It has a very good comment, too!